modules/gdscript/gdscript.h

```diff
@@ -196,6 +196,7 @@ class GDScript : public Script {
 	void clear(GDScript::ClearData *p_clear_data = nullptr);
 
 	virtual bool is_valid() const override { return valid; }
+	virtual bool is_abstract() const override { return false; /* false until #67777 gets merged */ }
```

Comment:
Not sure if we want to add a comment that references a specific PR https://github.com/godotengine/godot/pull/67777, but there's no issue or proposal AFAIK.